content/python/concepts/math-module/terms/math-atanh/math-atanh.md

```diff
@@ -0,0 +1,64 @@
+---
+Title: 'math.atanh()'
+Description: 'Returns the inverse hyperbolic tangent of a number'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.atanh()`** returns the inverse hyperbolic tangent of a number.
+
+## Syntax
+
+```pseudo
+math.atanh(x)
+```
+
+Since it is used to reverse the effect of a hyperbolic tangent function, the parameter `x` is always a number between -1 and 1. A `ValueError` will occur if:
+- `x` is greater than or equal to 1.
+- `x` is  less than or equal to -1.
+
```

Comment:
Remove line 26

Reply:
Line 26 now has code after the changes, so I removed line 27 instead